### PR TITLE
chore: Replace "DataOps" with "CLI for ELT+" in 2 more places, and update description.

### PR DESCRIPTION
**Pull request by DouweM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 <h1 align="center">Meltano - Your CLI for ELT+</h1>
-<h3 align="center">Open source, flexible, scales to your needs. Confidently move, transform, and test your data using tools you know with a data engineering workflow you’ll love.</h3>
+<h3 align="center">Open source, Flexible, and Scalable. Move, transform and test your data with confidence using a streamlined data engineering workflow you’ll love.</h3>
```
Comment (WillDaSilva) [inline README.md:3]:
```suggestion
<h3 align="center">Open source, Flexible, and Scalable. Move, transform, and test your data with confidence using a streamlined data engineering workflow you’ll love.</h3>
```

**Review by edgarrmondragon — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -53,11 +53,9 @@
 
 ---
 
-Welcome to your CLI for ELT+. It's open source, flexible, scales to your needs. Confidently move, transform, and test your data using tools you know with a data engineering workflow you’ll love.
-
-If you're a fan, star the repo ⭐️. [Plus this month, every ⭐ on GitHub removes 2 lb/~1 kg of trash from our waterways 🌊](https://meltano.com/blog/extracting-trash-to-transform-our-waterways/)
-
+Welcome to your CLI for ELT+: Open Source, Flexible, and Scalable. Move, transform and test your data with confidence using a streamlined data engineering workflow you’ll love.
```
Comment (WillDaSilva) [inline README.md:56]:
```suggestion
Welcome to your CLI for ELT+: Open Source, Flexible, and Scalable. Move, transform, and test your data with confidence using a streamlined data engineering workflow you’ll love.
```

File: pyproject.toml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "meltano"
 version = "2.13.0"
-description = "Meltano: Your DataOps Platform Infrastructure"
+description = "Meltano is your CLI for ELT+: Open Source, Flexible, and Scalable. Move, transform and test your data with confidence using a streamlined data engineering workflow you’ll love."
```
Comment (WillDaSilva) [inline pyproject.toml:4]:
```suggestion
description = "Meltano is your CLI for ELT+: Open Source, Flexible, and Scalable. Move, transform, and test your data with confidence using a streamlined data engineering workflow you’ll love."
```

**Review by WillDaSilva — COMMENTED** [review]
Why is `Open Source, Flexible, and Scalable` in title case?

File: README.md (hunk 1 above)
Comment (tayloramurphy) [reply README.md:3]: Oxford comma is a must!

**Review by afolson — APPROVED** [review]

**Review by WillDaSilva — APPROVED** [review]

**Pull request merged**